lib/perl/Genome/InstrumentData/AlignmentResult/Merged.pm

```diff
@@ -191,7 +196,9 @@ sub create {
         } else {
             #just collect the BAMs for a merge
             for my $alignment (@alignments) {
-                push @bams_for_final_merge, $alignment->alignment_bam_file_paths;
+                my $temp_allocation = $self->_get_temp_allocation($alignment);
```

Comment:
I want to pull it out... but in our first attempt to do so we added more lines of code than we took out :p. So I'm not sure if it can be made more clean. I will look again but let me know if you have an idea. We may leave it as-is.